src/style.css

```diff
@@ -1,3 +1,63 @@
+html {
+  margin: 0;
```

Comment:
The ‘margin: 0;’ and ‘padding: 0;’ styles are already being declared in the body selector. These styles are not needed in the html selector.

Reply:
✅